scripts/utils.py

```diff
@@ -111,8 +111,11 @@ def __init__(
             )
 
     def __str__(self):
+        environment = self._environ.copy()
+        if "IDP_CLIENT_SECRET" in environment:
+            environment["IDP_CLIENT_SECRET"] = "***"
```

Comment:
Great!
We should probably add COGNITE_CLIENT_SECRET and TRANSFORMATION_CLIENT_SECRET here as well-

Reply:
Yes definitely, but think the whole authentication needs a refactoring. Should not be necessary to setup URLs.